hoot-core/src/main/cpp/hoot/core/cmd/DeriveChangesetCmd.cpp

```diff
@@ -98,11 +106,34 @@ class DeriveChangesetCmd : public BaseCommand
       "Deriving changeset for inputs " << input1 << ", " << input2 << " and writing output to " <<
       output << "...");
 
-    //use the same unknown1 status for both so that difference doesn't influence the comparison
+    const double changesetBuffer = ConfigOptions().getChangesetBuffer();
+    if (changesetBuffer > 0.0)
+    {
+      //allow for calculating the changeset with a slightly larger AOI than the default specified
+      //bounding box; useful in certain situations
+      Envelope convertBounds =
+        GeometryUtils::envelopeFromConfigString(ConfigOptions().getConvertBoundingBox());
+      convertBounds.expandBy(changesetBuffer, changesetBuffer);
+      conf().set(
+        ConfigOptions::getConvertBoundingBoxKey(),
+        GeometryUtils::envelopeToConfigString(convertBounds));
+    }
+
+    //some in these datasets may have status=3 if you're loading conflated data, so use
+    //reader.use.file.status and reader.keep.file.status if you want to retain that value
     OsmMapPtr map1(new OsmMap());
     loadMap(map1, input1, true, Status::Unknown1);
     OsmMapPtr map2(new OsmMap());
-    loadMap(map2, input2, true, Status::Unknown1);
+    loadMap(map2, input2, true, Status::Unknown2);
+
+    //we don't want to include review relations
+    boost::shared_ptr<TagKeyCriterion> elementCriterion(
+      new TagKeyCriterion(MetadataTags::HootReviewNeeds()));
+    RemoveElementsVisitor removeElementsVisitor(elementCriterion);
+    removeElementsVisitor.setRecursive(false);
+    map1->visitRw(removeElementsVisitor);
+    map2->visitRw(removeElementsVisitor);
+
```

Comment:
I was assuming that by the time the changeset was derived the user would have resolved all review relations and made the corresponding edits to the conflated data as they would want it written back to MapEdit.  So, all duplicates would have been handled already.  So, dropping the review relations there is just a way of showing that if you didn't do all that beforehand, you were out of luck.